crypto/ocsp/ocsp_test.cc

```diff
@@ -0,0 +1,216 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+#include <gtest/gtest.h>
+
+#include "openssl/ocsp.h"
+
+#include "../internal.h"
+
+static const uint8_t ocsp_response_der[] = {
```

Comment:
Any unhappy paths you can check? Are there more happy paths you need to verify? Plans to expand this?

Reply:
This is the "best" no-problem happy path that I took from s2n. I'm planning to add even more happy/unhappy tests from the s2n unit tests, but the functionality I've implemented so far is not sufficient enough to test upon those yet.